bugwarrior/config.py

```diff
@@ -236,14 +236,14 @@ def get_data_path(config, main_section):
     # the `_` subcommands properly (`rc:` can't be used for them).
     line_prefix = 'data.location='
 
-    env={
-        'PATH': os.getenv('PATH'),
-        'TASKRC': taskrc,
-    }
-    # If TASKDATA is set but empty, taskwarrior's data.location is empty.
-    taskdata = os.getenv('TASKDATA')
-    if taskdata:
-        env['TASKDATA'] = taskdata
+    # Take a copy of the environment and add our taskrc to it.
+    env = os.environ
```

Comment:
Just an FYI, but on some platforms, changing the values of `os.environ` will actually change the calling environment's environment variables.  You probably want to create a copy of this by doing something like `env = dict(os.environ)`

Reply:
Oh, oops! I thought this *did* make a copy, but I'm still new to Python and learning when things are assigning-by-value and assigning-by-reference. Thank you so much for spotting this!

Updated changes landing soon. :)